lib/flagsmith/sdk/models/flag.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Flagsmith
+  # Flag object
+  class Flag
+    include Comparable
+
+    attr_reader :enabled, :value, :default, :feature_name, :feature_id
+
+    def initialize(feature_name:, enabled:, value:, feature_id:, default: false)
+      @feature_name = feature_name
+      @feature_id = feature_id
+      @enabled = enabled
+      @value = value
+      @default = default
+    end
+
+    def enabled?
+      @enabled
+    end
+
+    alias is_default default
+
+    def <=>(other)
```

Comment:
I'm not entirely sure why this is necessary again? Is it for some sort of ordering logic perhaps?